PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -16,7 +26,24 @@ class HomeViewController: UIViewController {
 
     private var slates: [Slate]? {
         didSet {
-            collectionView.reloadData()
+            var snapshot = NSDiffableDataSourceSnapshot<HomeSection, HomeItem>()
+
+            guard let slates = slates else {
+                dataSource.apply(snapshot)
```

Comment:
This is equivalent to saying that the new data source is completely empty, which would then render "nothing", correct? If that's true, is there an edge case where there _was_ some data present, "something happened" to return no slates, and the previous content disappears?